scripts/cxg_remaster.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import time
+
+import numpy as np
+import psutil
+import tiledb
+
+"""
+Re-master an existing "X" tiledb array in a CXG
+Creates a new array with the same data but a different tdb schema
+"""
+
+X_extent = None
+X_name = None
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("cxg", type=str, help="cxg path")
+    parser.add_argument("--kind", choices=["sparse", "dense"], type=str, help="target: sparse or dense")
```

Comment:
I recommend that this script make the determination of what encoding (sparse, dense) to use, rather than asking the caller/user to specify.  It should use whatever the current best heuristic in use for the ingestion pipeline.

Reply:
resolved - added default 'auto' mode